react-frontend/src/containers/App/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 //import Nav from '../../components/Nav';
+import NavBar from '../../components/Nav/navbar.js';
 import Home from '../../components/Home/home.js';
+import '../../css/home.css';
 
 function App() {
   return (
     <div className="App">
       {/* <p>THIS IS A TEST APP</p>
       <Nav /> */}
+      <NavBar />
+      <div className="contentSpacer">&nbsp;</div>
```

Comment:
What is this? &nbsp;  also if content spacer is just white space, that should be handled through css on the nave bar.